src/In.ProjectEKA.HipService/Gateway/GatewayClient.cs

```diff
@@ -45,7 +45,8 @@ public virtual async Task<Option<string>> Authenticate()
                     Method = HttpMethod.Post,
                     Content = new StringContent(json, Encoding.UTF8, MediaTypeNames.Application.Json)
                 };
-
+                if (correlationId != null)
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=6116067)